tests/kafkatest/services/console_consumer.py

```diff
@@ -21,7 +21,7 @@
 
 from kafkatest.directory_layout.kafka_path import KafkaPathResolverMixin
 from kafkatest.services.monitor.jmx import JmxMixin, JmxTool
-from kafkatest.version import DEV_BRANCH, LATEST_0_8_2, LATEST_0_9, LATEST_0_10_0, V_0_10_0_0, V_0_11_0_0, V_2_0_0
+from kafkatest.version import DEV_BRANCH, LATEST_0_8_2, LATEST_0_9, LATEST_0_10_0, V_0_10_0_0, V_0_11_0_0, V_2_0_0, LATEST_3_7
```

Comment:
`LATEST_3_7` does not exist yet, we need to wait for 3.7.0 to be out before adding it.

Reply:
Do I understand it right that you want to merge this only after 3.7.0 is released?

Reply:
Actually since `DEV_VERSION` already points to 3.8, I should be able to introduce `LATEST_3_7` even if it's not released yet. I'll try that.